try {
  exportIteration("NOITERATION", "csv", 25);
} catch (Exception e) {
  //
  // This is the expected behaviour
  //
}

TEST1="\"US9\",\"multiple subscriptions access\",\"As a user, I want to be able to ...\",\"Defined\",\"1.0\"";
TEST2="\"US11\",\"user stories export - Wiki format 1\",\"As a user, I want to export the ...\",\"Defined\",\"1.0\"";
TEST3="\"US7\",\"stories export - \"\"CSV\"\"\",\"As a user, I want to export the ...\",\"Accepted\",\"5.0\"";

stories = exportIteration("TESTIT01", "csv", 25);
print(stories);
assert(stories.indexOf(TEST1)>=0);
assert(stories.indexOf(TEST2)>=0);
assert(stories.indexOf(TEST3)>=0);

stories = exportIteration("TESTIT02", "csv", 25);
assert(stories.length() == 0);